### PR TITLE
add jmx interface for getting the active and pending tasks in FlushManager

### PR DESCRIPTION
**Pull request by jixuan1989**
Using jmx interface, we can know how many tasks are pending to be flushed. It is helpful for tuning parameters..

### CODE REVIEW

**Comment by jixuan1989** [comment]
it is the same with run `show flush task info` on cli...

File: server/src/main/java/org/apache/iotdb/db/service/ServiceType.java [hunk 1]
```diff
@@ -33,7 +33,9 @@
   SYNC_SERVICE("SYNC ServerService", ""),
   PERFORMANCE_STATISTIC_SERVICE("PERFORMANCE_STATISTIC_SERVICE","PERFORMANCE_STATISTIC_SERVICE"),
   TVLIST_ALLOCATOR_SERVICE("TVList Allocator", ""),
-  FLUSH_SERVICE("Flush ServerService", "");
+
+  FLUSH_SERVICE("Flush ServerService",
+      generateJmxName("org.apache.iotdb.db.engine.pool", "Flush Manager"));
```
Comment (jt2594838) [inline server/src/main/java/org/apache/iotdb/db/service/ServiceType.java]: How about renaming the JmxNames of other services that are registered in JMX too?

**Comment by jt2594838** [comment]
👍 We may add more statistics like the number of submitted tasks and average/maximum task runtime here, but this is good for now.

File: server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java [hunk 2]
```diff
@@ -84,4 +84,12 @@ public int getCorePoolSize() {
...
 
   public abstract String getName();
+
+  public int getNumberOfWorkingTasks() {
+    return ((ThreadPoolExecutor)pool).getActiveCount();
+  }
+
+  public int getNumberOfPendingTasks() {
+    return ((ThreadPoolExecutor)pool).getQueue().size();
+  }
```
Comment (fanhualta) [inline server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java]: Actually , the method has the same function with `getWaitingTasksNumber` in line 68 of this class.

File: server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java [hunk 3]
```diff
@@ -84,4 +84,12 @@ public int getCorePoolSize() {
   public abstract void stop();
 
   public abstract String getName();
+
+  public int getNumberOfWorkingTasks() {
+    return ((ThreadPoolExecutor)pool).getActiveCount();
+  }
```
Comment (fanhualta) [inline server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java]: Actually, this method has the same function with `getActiveCnt` in line 64 of this class.

File: server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java (hunk 2 above)
Comment (jixuan1989) [reply server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java]: fixed

File: server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java (hunk 3 above)
Comment (jixuan1989) [reply server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java]: fixed

**Pull request merged**